app/src/docs/grid.mdx

```diff
@@ -0,0 +1,7 @@
+import { NPM } from './lib/NPM';
+
+# Grid
+
+## A light-weight grid system for responsive layouts.
+
+<NPM npm={props.npm} />
```

Comment:
This `NPM` component could be expanded beyond `react` as we roll out more framework support. It could display all the relevant packages, and paths to the component collection in question.